build/mac/download_hermetic_xcode.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env vpython3
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Copyright 2018 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import argparse
+import os
+import platform
+import plistlib
+import subprocess
+import sys
+import pkg_resources
+
+try:
+    from urllib2 import URLError
+except ImportError:  # For Py3 compatibility
```

Comment:
we use python 3, don't need to pull this stuff from the upstream impl.

Reply:
there was a lint issue around this and I was having trouble running pylint locally on my arm mac so I just copied from other scripts because I knew it would pass lint